src/bandersnatch/tests/test_mirror.py

```diff
@@ -11,9 +12,18 @@
 from bandersnatch.configuration import BandersnatchConfig, Singleton
 from bandersnatch.filter import filter_project_plugins
 from bandersnatch.mirror import Mirror
+from bandersnatch.package import Package
 from bandersnatch.utils import WINDOWS
 
 
+def touch_files(paths: List[Path]):
+    for path in paths:
+        if not path.parent.exists():
+            path.parent.mkdir(parents=True)
+        with path.open("wb") as pfp:
+            pfp.close()
```

Comment:
This is copied and pasted, which sucks, but it's only temporary until the other parts of ```Package``` are relocated as well.  I can move it somewhere and import from both places if you prefer.

Reply:
Can’t we just move to utils or a new test_utils.py to save the copy?